MetasysServices/Equipments/EquipmentServiceProvider.cs

```diff
@@ -89,14 +89,14 @@ public async Task<IEnumerable<MetasysPoint>> GetPointsAsync(ObjectId equipmentId
                     MetasysPoint point = new MetasysPoint(item);
                     // Retrieve object Id from full URL
                     string objectId = point.ObjectUrl.Split('/').Last();
-                    point.ObjectId = ParseObjectIdentifier(objectId);
+                    point.ObjectId = objectId;
                     // Retrieve attribute Id from full URL
                     string attributeId = (point.Attribute != null) ? point.Attribute.Split('.').Last() : String.Empty;
                     if (attributeId == String.Empty)
                     {
                         attributeId = (point.AttributeUrl != null) ? point.AttributeUrl.Split('/').Last() : String.Empty;
                     }
-                    if (point.ObjectId != Guid.Empty) // Sometime can happen that there are empty Guids.
+                    if (!string.IsNullOrWhiteSpace(point.ObjectId)) // Sometime can happen that there are empty Guids.
```

Comment:
Guid.Empty is a guid with all zeroes. Not sure how that could happen. I've never seen the API return that. But if it did happen then my fix doesn't replicate the same logic. 

If we need to really support that then I need to change this conditional to look for `"00000000-0000-0000-0000-000000000000"`

Reply:
I don't think this should be merged until we address this question @Federico-Artioli